src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
It's a good practice to explicitly compare the property to 'undefined' to check for its existence. This makes the intention clearer and the code more readable.